astropy_helpers/openmp_helpers.py

```diff
@@ -39,6 +41,34 @@
 }
 """
 
+def _split_option_from_var(option, var, delim=' '):
+    if not var or not option:
+        return None
+    l = len(option)
+    if not l:
+        return None
+
+    if var in os.environ:
+        for flag in os.environ.split(delim):
+            if option in flag[:l]:
+                return flag[l:]
+
+    # os.environ was unsuccesful so Look in sysconfig
+    try:
+        flags = get_config_var(var)
+    except KeyError:
+        return None
+    if flags:
+        for flag in flags.split(delim):
+            if option in flag[:l]:
+                return flag[l:]
+    return None
+
+def _get_include_path():
+    return _split_option_from_var('-I', 'CFLAGS')
+
+def _get_library_path():
+    return _split_option_from_var('-L', 'LDFLAGS')
 
 def add_openmp_flags_if_available(extension):
```

Comment:
Add a note in the docstring that one can pass ``None`` to find out if OpenMP compilation would work without doing it for a specific extension.

Reply:
Will do. Maybe it's also a good idea to make it the default, `extension=None`, and also add a "wafer thin" wrapper with a more semantically correct name?

Reply:
Should probably suppress the exception when `None` also(?)

Reply:
I don't know if it's super important to have the semantically correct name, you can always just put a comment above the call to the function to explain what it does in that case?